scripts/test_nearlib.sh

```diff
@@ -1,12 +1,16 @@
 #!/usr/bin/env bash
 set -ex
 
+SRC_DIR=$(pwd)/$(dirname "${BASH_SOURCE[0]}")
+export REPO_DIR=$(readlink -e "$SRC_DIR/..")
+pushd "${REPO_DIR}"
```

Comment:
This is really confusing. First of all, no need to `export`, just `repo_dir=whatever`. Second of all, why `pushd`? Just `cd`. And lastly, the name resolution doesn’t seem correct. I think what you want is:

    repo_dir=$(realpath -e "$(dirname "${BASH_SOURCE[0]}")/..")

Reply:
`push` / `popd`, show which commands need to be executed in a specific directory.

This is a good example:
```
# Run near-api-js tests
pushd "${repo_dir}"/near-api-js
yarn
yarn build
"${repo_dir}"/scripts/waitonserver.sh
popd
```

Reply:
I’d just `cd` each time. I’d say that’s more readable than having to maintain directory stack in your head when you read the script.